assets/sass/patterns/atoms/list.scss

```diff
@@ -18,3 +18,7 @@ $prefixes: ("alpha-lower": "lower-latin", "alpha-upper": "upper-latin", "bullet"
     }
   }
 }
+
+.list--teaser {
+  @include padding($blg-space-small-in-px, "top");
```

Comment:
Is this meant to be separating the list from the text? Or are there more requirements not yet fulfilled? If it's just this, can the text just be wrapped in a `<p>` instead?